hotsos/defs/scenarios/juju/juju_pebble_cve.yaml

```diff
@@ -0,0 +1,63 @@
+checks:
+  has_affected_pebble:
+    snap:
+      pebble:
+        - min: 646
+          max: 646
+    apt:
+      pebble:
+        - min: 1.4.0
+          max: 1.4.1
+        - min: 1.7.0
+          max: 1.7.3
+        - min: 1.10.0
+          max: 1.10.1
+  has_2_9_affected_juju:
```

Comment:
Since the CVE actually only affects Pebble (affects Juju only as a dependency), wouldn't it be sufficient to check the Pebble versions (as above)?

Reply:
juju also needed to be patched. I think pebble is a library dependency.

Reply:
Right. I checked this a bit. It seems there's no way to check "if this Juju is using pebble x". So I think this is fine. But I think we should take out the pebble bits out of this check and add it as a separate scenario as Pebble could be used independent of Juju for something else (also it needs to cover snap install too along with `apt`).